### DISCUSSION TITLE
Blending Per (Groups of) Layer(s)

### DISCUSSION THREAD

**Discussion by ilan-gold** [visgl/deck.gl#5450] [Q&A] [answered]
Hello,

Is it possible to blend the "top" n layers one way and the "bottom" n layers another way?  My assumption has always been "no" since to my knowledge the blending is controlled by `gl.blendFunc` per WebGL canvas.

**Comment by Pessimistress** [comment, marked as answer]
You can set per-layer blending using `props.parameters`:

```js
import GL from '@luma.gl/constants';

new ScatterplotLayer({
  // ...
  parameters: {
    blendFunc: [GL.ONE, GL.ONE, GL.ONE, GL.ONE],
    blendEquation: GL.FUNC_ADD
  }
})
```

**Reply by ilan-gold** [reply]
Thanks!
